google/api_core/client_info.py

```diff
@@ -39,9 +39,9 @@
 class ClientInfo(object):
     """Client information used to generate a user-agent for API calls.
 
-    This user-agent information is sent along with API calls to allow the
-    receiving service to do analytics on which versions of Python and Google
-    libraries are being used.
+    This user-agent information is sent along with API calls via the
+    `x-goog-api-client` header to allow the receiving service to do analytics
```

Comment:
Not strictly accurate. I believe this client info actually does populate `user-agent`. It's the grpc version that populates `x-goog-api-client`.

Reply:
Good catch! I'll update the wording so that users are aware when to expect a `user-agent` header

Reply:
Moving forward, only the `x-goog-api-client` will contain the `user-agent` information. We won't be populating `user-agent` as a separate header. I suggest we keep the wording as is.